R/sumofsquares.R

```diff
@@ -18,7 +19,7 @@ SumOfSquares <- function(x, weights = NULL)
         return(ss)
     }
     if (nrow(x) != length(weights))
-        stop("The number of rows in 'x' does not match the length 'weights and ")
+        StopForUserError("The number of rows in 'x' does not match the length 'weights and ")
```

Comment:
Is there an error in the message?

Reply:
yeah its incomplete. I could delete the extra and?

Reply:
I think so, and add a single quote to the end of `weights` and full stop.